recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/doma-generic-h3-4x2g.cfg

```diff
@@ -4,7 +4,7 @@ seclabel='system_u:system_r:domU_t'
 name = "DomA"
 
 # Kernel image to boot
-kernel = "/xt/doma/Image"
+kernel = "/xt/doma/u-boot.bin"
```

Comment:
I fail to see the code that will install u-boot at this location

Reply:
fixed, see u-boot_%.bbappend